types/wellknown/wellknown-tests.ts

```diff
@@ -8,3 +8,21 @@ const geoJson: wellknown.GeoJSONGeometry = {
 };
 
 wellknown.stringify(geoJson); // $ExpectType string
+
+wellknown.parse("GEOMETRYCOLLECTION (1 2)"); // $ExpectType GeoJSONGeometry
```

Comment:
I haven't worked with WKT strings for a while but shouldn't this be something like `GEOMETRYCOLLECTION( POINT(1, 2))` ?

Reply:
...yes, yes it should. 🤦  Thanks, fixed.